titleids.txt

```diff
@@ -13,6 +13,7 @@ DKC: Tropical Freeze | 0005000010137F00, 0005000010138300, 0005000010144800
 DuckTales: Remastered | 0005000010129000, 0005000010129200
 FAST Racing NEO | 000500001012F000, 00050000101D6000, 00050000101E4100, 00050000101FED00
 FotNS: Ken's Rage 2 | 0005000010116600, 000500001012B800, 000500001012B900
+Friends | 0004013000003202, 000500301001500A, 000500301001510A, 000500301001520A
```

Comment:
The title IDs should be added inside `build_games_list.js`, like we are doing with the other params of the friends server

Reply:
They are though

Reply:
No, I mean hardcoding the title IDs inside `build_games_list.js`, like we do with other information: https://github.com/PretendoNetwork/nex-viewer/blob/465b72a456db80e00bb0948cbe6e1bca6252b613/build_games_list.js#L20-L29

Reply:
They are though

Reply:
This is what I meant [`8bcbb28` (#15)](https://github.com/PretendoNetwork/nex-viewer/pull/15/commits/8bcbb28fa6024c607fcb49ccd3f65ca4ddd7d5ef)

Reply:
Oh

Reply:
I did not notice that at all :skull:

Why is it hardcoded?